client/package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-react": "6.8.0",
+    "expect": "^1.20.2",
```

Comment:
I have no idea. I was getting an error in my build and I went to check the version of `expect` we were using, and found that it wasn't there. If any other projects list it as a dependency, we would be able to use it as a side effect.